api/v1/helpers.go

```diff
@@ -462,6 +462,11 @@ func (s *Subcluster) GetVProxyConfigMapName(vdb *VerticaDB) string {
 	return fmt.Sprintf("%s-%s-proxy-cm", vdb.Name, s.Name)
 }
 
+// GetVProxyConfigMapName returns the name of the client proxy config map
+func (v *VerticaDB) GetVProxyConfigMapName(scName string) string {
```

Comment:
There is already `names.GenVProxyConfigMapName`, what do you need this for?

Reply:
The names.GenVProxyConfigMapName takes params: vdb *vapi.VerticaDB, sc *vapi.Subcluster
But when the time I need to generate the configMapName, I don't have the sc object available.
Only the subclusterName, which is retrieved from sts label.